cmd/update.go

```diff
@@ -55,26 +55,19 @@ func newUpdateCmd() *updateCmd {
 
 			// Update single binary
 			if bin != "" {
-				// If user enter a command update like this:  $ bin update yq
-				// Find path on the config file
 				if !strings.Contains(bin, "/") {
-					for _, b := range cfg.Bins {
-						if b.RemoteName == bin {
-							bin = b.Path
-							break
-						}
+					var err error
+					bin, err = getBinPath(bin)
+					if err != nil {
+						return err
 					}
 				}
+				b := cfg.Bins[bin]
 
-				if b, found := cfg.Bins[bin]; !found {
-					return fmt.Errorf("Binary path %s not found", bin)
```

Comment:
This is now handled by `getBinPath`.

Reply:
shouldn't `getBinPath` also contains the `if !strings.Contains` condition as well? 

I thought we could move the whole `if` block to that function.